addons/web/static/src/legacy/scss/data_export.scss

```diff
@@ -8,8 +8,10 @@
 }
 
 @include media-breakpoint-down(sm) {
-    .ui-sortable-handle {
-        display: none;
+    .o_export {
+        .ui-sortable-handle {
```

Comment:
could need validation from someone at mobile team like @pparidans or @adr-odoo maybe ? I have no idea of the impact it could have..

Reply:
Sorting things up is tricky on touch screens, even more on smaller ones (like smartphones...) ; hence disabling/hiding this kind of features.

In the current state of Odoo, switching from one breakpoint to another (specifically between the "mobile" one and an upper one) still requires a refresh. Also the boundary between "small screen" and "mobile device" is blurry... But this will eventually change as our codebase goes more and more to a "responsive" way of dealing with this.